core/domain/html_validation_service.py

```diff
@@ -705,3 +705,156 @@ def convert_svg_diagram_tags_to_image_tags(html_string):
+        (u'\xe0\xa3', u'\xe3'),
+        (u'\xe0\xa1', u'\xe1'),
+        (u'\xe0\xa2', u'\xe2'),
+        (u'\xe0\xa4', u'\xe4'),
+        (u'\xe0\xa5', u'\xe5'),
+        (u'\xe0\xa6', u'\xe6'),
+        (u'\xe0\xa7', u'\xe7'),
+        (u'\xe0\xa8', u'\xe8'),
+        (u'\xe0\xa9', u'\xe9'),
+        (u'\xe0\xaa', u'\xea'),
+        (u'\xe0\xab', u'\xeb'),
+        (u'\xe0\xac', u'\xec'),
+        (u'\xe0\xad', u'\xed'),
+        (u'\xe0\xae', u'\xee'),
+        (u'\xe0\xaf', u'\xef'),
+        (u'\xe0\xb1', u'\xf1'),
+        (u'\xe0\xb3', u'\xf3'),
+        (u'\xe0\xba', u'\xfa'),
+        (u'\xe0\xbb', u'\xfb'),
+        (u'\xe0\xbd', u'\xfd'),
+        (u'\xe0\xb5', u'\xf5'),
+        (u'\xe0\xb6', u'\xf6'),
+        (u'\xe0\xb7', u'\xf7'),
+        (u'\xe0\xbc', u'\xfc'),
+        (u'\xe0\u2022', u'\xd5'),
+        (u'\xe0\u2021', u'\xc7'),
+        (u'\xe0\u2013', u'\xd6'),
+        (u'\xe0\u2018', u'\xd1'),
+        (u'\xe0\u201c', u'\xd3'),
+        (u'\xe0\u201e', u'\xc4'),
+        (u'\xe0\u20ac', u'\xc0'),
+        (u'\xe0\u0153', u'\xdc'),
+        (u'\xe0\u2014', u'\xd7'),
+        (u'\xe0\u0178', u'\xdf'),
+        (u'\xc4\u0178', u'\u011f'),
+        (u'\xc4\xab', u'\u012b'),
+        (u'\xc4\xbb', u'\u013b'),
+        (u'\xc4\xb1', u'\u0131'),
+        (u'\xc4\xb0', u'\u0130'),
+        (u'\xc4\u2021', u'\u0107'),
+        (u'\xc4\u2122', u'\u0119'),
+        (u'\xc4\u2026', u'\u0105'),
+        (u'\xc4\u20ac', u'\u0100'),
+        (u'\xc4\xb0', u'\u0130'),
+        (u'\xc5\xba', u'\u017a'),
+        (u'\xc5\xbe', u'\u017e'),
+        (u'\xc5\u203a', u'\u015b'),
+        (u'\xc5\u0178', u'\u015f'),
+        (u'\xc9\u203a', u'\u025b'),
+        (u'\xd9\u2026', u'\u0645'),
+        (u'\xd1\u02c6', u'\u0448'),
+        (u'\xd8\xb5', u'\u0635'),
+        (u'\xd8\xad', u'\u062d'),
+        (u'\xd8\xa4', u'\u0624'),
+        (u'\xe1\xba\xbf', u'\u1ebf'),
+        (u'\xe1\xbb\u0178', u'\u1edf'),
+        (u'\xe2\u20ac\u0153', u'\u201c'),
+        (u'\xe2\u02c6\u2030', u'\u2209'),
+        (u'\xe2\u2026\u02dc', u'\u2158'),
+        (u'\xe2\u02c6\u0161', u'\u221a'),
+        (u'\xe2\u02c6\u02c6', u'\u2208'),
+        (u'\xe2\u2014\xaf', u'\u25ef'),
+        (u'\xe2\u20ac\u201c', u'\u2013'),
+        (u'\xe2\u2026\u2013', u'\u2156'),
+        (u'\xe2\u2026\u201d', u'\u2154'),
+        (u'\xe2\u2030\xa4', u'\u2264'),
+        (u'\xe2\u201a\xac', u'\u20ac'),
+        (u'\xe3\u201a\u201e', u'\u3084'),
+        (u'\xe3\u201a\u201c', u'\u3093'),
+        (u'\xe3\u201a\u201a', u'\u3082'),
+        (u'\xe3\u201a\u2019', u'\u3092'),
+        (u'\xe3\u201a\u0160', u'\u308a'),
+        (u'\xe5\u0152\u2014', u'\u5317'),
+        (u'\xe6\u0153\xa8', u'\u6728'),
+        (u'\xe6\u02c6\u2018', u'\u6211'),
+        (u'\xe6\u02dc\xaf', u'\u662f'),
+        (u'\xe8\xa5\xbf', u'\u897f'),
+        (u'\xe9\u201d\u2122', u'\u9519'),
+        (u'\u7aef', u'\xfc'),
+        (u'\u8d38', u'\xf3'),
+        (u'\ucc44', u'\xe4'),
+        (u'\uccb4', u'\xfc'),
```

Comment:
Please add:

-        (u'\u00f3', u'\xf3'),
-        (u'\u00a0', u'\xa0'),
-        (u'\u00ed', u'\xed'),
-        (u'\u00e7', u'\xe7'),
-        (u'\u00e3', u'\xe3'),
-        (u'\u00e9', u'\xe9'),
-        (u'\u00f5', u'\xf5'),
-        (u'\u00a0', u'\xa0'),
-        (u'\u00c9', u'\xc9'),
-        (u'\u00e0', u'\xe0'),

Reply:
I've updated based on the latest list!